docs/api/utility.rst

```diff
@@ -0,0 +1,325 @@
+.. # Copyright (c) 2017-2019, Lawrence Livermore National Security, LLC and
+.. # other BLT Project Developers. See the top-level COPYRIGHT file for details
+.. # 
+.. # SPDX-License-Identifier: (BSD-3-Clause)
+
+Utility Macros
+==============
+
+
+General Utility Macros
+----------------------
+
+
+blt_list_append
+~~~~~~~~~~~~~~~
+
+.. code-block:: cmake
+
+    blt_list_append( TO <list> ELEMENTS [ <element>...] IF <bool> )
```

Comment:
I think it would be easier to read these macro prototypes if each argument was on its own line.

Reply:
fixed.